modules/ROOT/pages/index.adoc

```diff
@@ -82,6 +82,134 @@ Monitor APIs and endpoints at scheduled intervals. Receive reports about whether
 
 See xref:api-functional-monitoring::afm-in-anypoint-platform.adoc[API Functional Monitoring in Anypoint Monitoring].
 
+== Anypoint Monitoring Features
+
+Anypoint Monitoring provides many features to Gold and Platinum subscribers. However, for more comprehensive monitoring services, Titanium subscribers receive additional charts and functionality.
```

Comment:
We need to talk about Titanium in a more enticing way, and should be aligned with our marketing message. There are quite a few Bloomfire marketing materials that was included in the trust plan doc, please incorporate the materials here. As an example, we should use words like "advanced" when describing Titanium. 

https://docs.google.com/document/d/1YNMX7kAVl2YCLO_hfx2Rlf5SmHV1gk9O0CH5tJ2RmPQ/edit#

Reply:
Anypoint Monitoring is the standard method of monitoring Mule application performance, logging, and troubleshooting to reduce the mean time to identify issues and resolve them. While all customers receive fundamental monitoring capabilities with their subscription, the Titanium tier offers greater storage, single-tenancy and advanced capabilities that allows for greater control and flexibility.

Reply:
Don't use the word "single-tenancy".  Rather use the word "guaranteed resources".  Single tenancy is only used in some parts of the system such as storage.  Other parts of the system is not single tenanted.  As an example, we do not use single tenancy for alert processing, UX, etc. for Titanium.

Reply:
```suggestion
Anypoint Monitoring is the standard method of monitoring Mule application performance, logging, and troubleshooting to reduce the mean time to identify issues and resolve them. While all customers receive fundamental monitoring capabilities with their subscription, the Titanium tier offers greater storage, guaranteed resources, and advanced capabilities that allow for greater control and flexibility.
```